File: bestia-models/src/main/java/net/bestia/model/dao/AttackDAO.java
package net.bestia.model.dao;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;
import org.springframework.transaction.annotation.Transactional;

import net.bestia.model.domain.AttackImpl;

@Repository("attackDao")
@Transactional(readOnly = true)
public interface AttackDAO extends CrudRepository<AttackImpl, Integer> {

}

File: bestia-models/src/main/java/net/bestia/model/dao/AttackDAO.kt
package net.bestia.model.dao

import org.springframework.stereotype.Repository
import org.springframework.transaction.annotation.Transactional

import net.bestia.model.domain.AttackImpl
import org.springframework.data.repository.CrudRepository

@Repository("attackDao")
@Transactional(readOnly = true)
interface AttackDAO : CrudRepository<AttackImpl, Int>
